sql-cli/sql_cli/utils/rich.py

```diff
@@ -0,0 +1,29 @@
+from typing import IO, Any, Optional
+
+from typer.rich_utils import _get_rich_console
+
+
+def rprint(
```

Comment:
Should we also use this function as an import instead of
1. https://github.com/astronomer/astro-sdk/blob/f50be8d590ff29b44b81e3ba340152bc316a429b/sql-cli/sql_cli/run_dag.py#L19

2. https://github.com/astronomer/astro-sdk/blob/f50be8d590ff29b44b81e3ba340152bc316a429b/sql-cli/sql_cli/connections.py#L7

Reply:
Yes, makes sense. Good catch! 👍